templates/docs/examples/layouts/application/structure.html

```diff
@@ -4,6 +4,17 @@
 {% block style %}
 <style>
   body { margin: 0; }
+
+  /* example structure styling to make panels visible */
+  .app-demo > .l-navigation-bar,
+  .app-demo > .l-main,
+  .app-demo > .l-navigation .l-navigation__drawer,
+  .app-demo > .l-aside,
+  .app-demo > .l-status {
+    padding: 0.5rem;
+    background-color: #fff;
+    border: 1px solid #d9d9d9;
```

Comment:
linting/alphabetical order:
```suggestion
    background-color: #fff;
    border: 1px solid #d9d9d9;
    padding: 0.5rem;
```